### DISCUSSION TITLE
Proposta d'Esame Manuel Bacci

### DISCUSSION THREAD

**Discussion by ManuelBax** [deneb-alpha/Lab_open_source-2024_2025#21] [Projects]
#Task richiesti
Introduzione al concetto di Open Hardware
Presentazione di un progetto open hardware sviluppato con Arduino
descrizione tecnica del progetto
#Task Opzionali
Concetto di Open Source
Contributo alla documentazione di libre office
Confronto  Microsoft Office e LibreOffice

**Comment by deneb-alpha** [comment]
Ciao @ManuelBax 
per quanto riguarda la documentazione di LibreOffice, il primo passo da fare è presentarsi nel [gruppo telegram del team di localizzazione in italiano](https://t.me/localizzazione_libreoffice) e poi verificare con i coordinatori da dove iniziare in base alle priorità correnti.

Per il progetto d'esame, puoi presentare il lavoro svolto e anche descrivere come hai interagito con il gruppo di localizzazione.

**Comment by ManuelBax** [comment]
Salve professoressa ho aggiornato la proposta d'esame, mi interesserebbe portare un progetto open hardware sviluppato con arduino e trattare il tema dell'hopen hardware, per quanto riguarda la documentazione l'ho messa come task aggiuntivo ricollegato poi il tema dell open source e la discussione tra office 365 e libre office, in caso pensavo di contattare Gabriele su linkedin e riguardo la documentazione il gruppo che mi suggeriva lei, e magari se possibile italo per la discussione di lobreoffice e microsoft office, rimango in attesa, buona giornata

Inviato da Outlook per Android<https://aka.ms/AAb9ysg>
________________________________
